### PR TITLE
[MRG, ENH] Add montage (R-Net 128 Ch from Brain Products)

### PR DESCRIPTION
**Pull request by JungleHippo**
#### What does this implement/fix?
This adds the montage of R-Net 128 Ch from Brain Products (RNP-BA-128) to the standard built-in montages.
#### Additional information
Also adds the respective test lines, pytest run without errors.

### Ready for review

### CODE REVIEW

**Comment by drammock** [comment]
there is a code-style failure:
https://dev.azure.com/mne-tools/mne-python/_build/results?buildId=20909&view=logs&j=c1ec3add-0fcd-52ea-530a-328457bbfb2e&t=056d958c-96b0-5ff4-dcb2-798466eed616&l=15

Other than that, this looks like a good addition. Since we don't have a place in our docs where we show *all* the built-in montages ([this tutorial](https://mne.tools/stable/auto_examples/visualization/montage_sgskip.html) is intentionally not run by our server because it takes too long), could you just post a screenshot / saved image of what this montage looks like when plotted, as a sanity check that it looks reasonable?

**Comment by drammock** [comment]
also, please add a changelog entry to `doc/changes/latest.inc` under "enhancements", something like
```
- Added new built-in montage XXX (:gh:`10974` by :newcontrib:`Ilias Machiaras`)
```
and then add yourself to `doc/changes/names.inc` as well.

**Comment by JungleHippo** [comment]
I am attaching the image, the additions to docs are also done. 

Sensor_positions_(eeg)_RNP_BA_128

File: doc/changes/latest.inc [hunk 1]
```diff
@@ -206,7 +208,7 @@ Bugs
 - When reading EGI MFF files, channel names are no longer ignored and reference channel information is properly incorporated (:gh:`10898` by `Scott Huberty`_ and `Daniel McCloy`_)
 
 - Fix bug in :class:`mne.decoding.TemporalFilter` where filter parameters were not handled properly (:gh:`10968` by `Eric Larson`_)
-- 
+-
```
Comment (drammock) [inline doc/changes/latest.inc:211]:
```suggestion

```

File: mne/channels/_standard_montage_utils.py [hunk 2]
```diff
@@ -163,6 +163,8 @@ def _mgh_or_standard(basename, head_size, coord_frame='unknown'):
                                basename='artinis-octamon.elc'),
     'artinis-brite23': partial(_mgh_or_standard, coord_frame='mri',
                                basename='artinis-brite23.elc'),
+    'brainproducts-RNP-BA-128': partial(_easycap,
+                                basename='brainproducts-RNP-BA-128.txt')
```
Comment (drammock) [inline mne/channels/_standard_montage_utils.py:167]:
```suggestion
    'brainproducts-RNP-BA-128': partial(
        _easycap, basename='brainproducts-RNP-BA-128.txt')
```

**Review by drammock — APPROVED** [review]

**Comment by drammock** [comment]
@JungleHippo the style error was still there, so I fixed it through the GitHub review interface. Let's see if the CI tests come back green!

**Pull request merged**